fuel-tx/src/transaction/consensus_parameters/gas/default_gas_costs.rs

```diff
@@ -161,5 +161,6 @@ pub fn default_gas_costs() -> GasCostsValues {
             base: 44,
             dep_per_unit: 5,
         },
+        contract_root: 3024932,
```

Comment:
Hmm, it is a very huge value. Maybe it is better if we use `DependentCost` instead. Could you check how expensive it is in this case, please?